scripts/characters/races.gd

```diff
@@ -0,0 +1,789 @@
+
```

Comment:
This file alters the stats, pricemods, and descriptions(they key part being the stats) of most races without condition. Also the tabbing regressed in stats.

Reply:
I'll come back to this one

Reply:
Man this ones gonna be a pain in the ass.